test/module/shared_model/backend_proto/shared_proto_transaction_test.cpp

```diff
@@ -75,20 +75,18 @@ iroha::protocol::AddAssetQuantity generateAddAssetQuantity(
  * @then transaction is built correctly
  */
 TEST(ProtoTransaction, Builder) {
-  uint64_t created_time = 10000000000ull;
-  shared_model::interface::Transaction::TxCounterType tx_counter = 1;
-  std::string account_id = "admin@test", asset_id = "coin#test",
-              amount = "10.00";
-
   iroha::protocol::Transaction proto_tx = generateEmptyTransaction();
+
+  std::string account_id = "admin@test", asset_id = "coin#test",
```

Comment:
Same for this about the scope

Reply:
No, here we put valid account_id and asset_id. In next test we use invalid ones, to check if stateless validation fails

Reply:
Ah, I see. It's fine then